css/index.css

```diff
@@ -0,0 +1,56 @@
+
+body
+{
+    background-color: lightblue; 
+    color: red;
+}
+       section{
+        margin-bottom: 100px;
+       }
+
+       h1{
+        font-size: medium;
+        font-weight: 200;
+        color: black;
+        font-style: italic;
+       }
+
+       h3{
+        font-size: medium;
+        font-weight: 200;
+        color: yellow;
+        font-style: normal;
+       }
+
+       h4{
+        font-style:inherit;
+       }
+
+       a{
+        color: brown;
+       }
+
+       
+       a:visited{
+        color: black;
+                        
+       }
+
+       a:active{
+        color: blue;
+       }
```

Comment:
For the CSS to work properly, the links in the "Connect" section of the HTML must be correct.